packages/fuels-accounts/src/predicate.rs

```diff
@@ -41,10 +41,16 @@ impl Predicate {
 
     pub fn load_from(file_path: &str) -> Result<Self> {
         let code = fs::read(file_path).map_err(|e| {
-            error!(
-                IO,
-                "could not read predicate binary {file_path:?}. Reason: {e}"
-            )
+            match path::absolute(file_path).map(path_clean::clean) {
+                Ok(clean_file_path) => error!(
+                    IO,
+                    "could not read predicate binary {clean_file_path:?}. Reason: {e}",
+                ),
+                Err(e) => error!(
```

Comment:
What would be ideal IMO is the following:
 read the original path - success - > all good
 read the original path -> fails -> clean it up - cleanup successful -> report nice err message
 read the original path -> fails -> clean it up -> cleanup fails -> report original err message without mention of the cleanup failing